packages/blocks/src/answer/index.js

```diff
@@ -20,9 +22,20 @@ const Input = styled.input`
 const Answer = ( { attributes, className } ) => {
 	const width = attributes.width ? `${ attributes.width }%` : null;
 
+	const parentQuestion = useContext( MultipleChoice.Context );
+
+	// REVIEWER NOTE: choiceType value is assigned from ChoiceType constant under block-editor/multiple-choice/constants,
```

Comment:
`@crowdsignal/block-editor` is dependent on `@crowdsignal/blocks` so whenever dealing with these things, I believe this is the place they should live.  
The idea behind that is: we want to be able to build widget bundles and not include the editor stuff at all.

The `./constants.js` files in `block-editor` are currently kind of a left-over. They were copied there straight from `crowdsignal-forms` just to get the blocks to register in the editor, but now that we have the two packages they belong in `@crowdsignal/blocks`.

To import it inside the block-editor package, I think it's best to make it a property of `MultipleChoice`. So something like this:

```
import { ChoiceType } from `./constants';

const MultipleChoice = ( props ) => {
    ...
};

MultipleChoice.ChoiceType = ChoiceType.

export default MultipleChoice;
```

That said, I don't think there's a need for a `ChoiceType` constant altogether.  
Like we discussed, we need a `minimum/maximum choices` restriction as well. While I'm not sure how exactly we're going to enforce it yet these conditions are absolutely equivalent:

```
(minimumChoices === 1 && maximumChoices === 1) === ChoiceType.SINGLE
```

So I'd just go ahead and use the former and have one less thing to worry about.

Reply:
Thanks! That sounds good.

The ChoiceType thing was derived from our backend classes as a 1:1 correlation, but it's true that we could just rely on max/min and work from there. Mind that, if we go with this "implying" mechanism, a value of `min = 1` would also state the question as mandatory. I think that's the only reason the `type` switch existed: to keep those concepts apart on the UI.

I'll change things and ping you.